src/app/main.ts

```diff
@@ -34,24 +36,13 @@ const setAboutPanel = (clientVersion: string, buildNumber: string) => {
     });
 };
 
-if (isMac) {
-    let { clientVersion, buildNumber }: IVersionInfo = versionHandler.getVersionInfo();
-    setAboutPanel(clientVersion, buildNumber);
-    eventEmitter.on('update-version-info', (versionInfo) => {
-        clientVersion = versionInfo.clientVersion;
-        buildNumber = versionInfo.buildNumber;
-        setAboutPanel(clientVersion, buildNumber);
-    });
-}
-
-// Electron sets the default protocol
-app.setAsDefaultProtocolClient('symphony');
-
 /**
  * Main function that init the application
  */
 const startApplication = async () => {
     await app.whenReady();
+    const versionInfo: IVersionInfo = await versionHandler.getClientVersion();
```

Comment:
This will prevent the application from launching if there any network related issues

Reply:
Thanks Kiran for the observation. I've corrected it in the next commit.